packages/terra-form/docs/Input.md

```diff
@@ -0,0 +1,16 @@
+# Input
+
+Generic input which represents an HTML input element directly.
+
+## Usage
+
+```jsx
+import React from 'react';
+import { Input } from 'terra-form';
+
+<Input
+  required={false}
```

Comment:
Nit: Our Booleans are getting a little awkward. This is kind of up to preference, this value defaults to false, so passing in an explicit false does the same thing. There are also examples in these docs that use `required={true}` when just passing required in should do the same thing. Do we have a preference or standard on how we recommend using booleans? The two generic paths being:
```
<Input required />
```
or
```
<Input required={true} />
```
The current prop `required` also breaks our `is` prefix, but it matches the html API, so I don't know which one is more user friendly